### PR TITLE
Post and get feature

### PR DESCRIPTION
**Pull request by StormTrooper-007**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -104,7 +104,7 @@ jobs:
         uses: jtalk/url-health-check-action@v3
         with:
           #Set App Name (replace "example" with "alpha"-"tango")
-          url: http://weatherwise.capstone-project.de
+          url: https://cgn-java-23-2-collins.capstone-project.de/
           max-attempts: 3
           retry-delay: 5s
           retry-all: true
```
Comment (Pia0912) [inline .github/workflows/deploy.yml]: The reason for the mean red icon is, that you are missing an empty line at the end of this file 😃

File: frontend/src/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export type Todo = {
+    id: string
+    plan: string
+    startTime: string
+    status: string;
+    createdAt: string;
+}
```
Comment (Pia0912) [inline frontend/src/utils.tsx]: Here you can also add an empty line 😀

**Review by Pia0912 — APPROVED** [review]
LGTM! SonarCloud says you are missing a test (62.7% Coverage for your ToDo model) I had the same issue & Marcell sent me this: "Du kannst Lombok von Sonar excluden: https://stackoverflow.com/questions/47430397/excluding-lombok-classes-from-sonar-coverage-report"  -> add lombok.config to your backend (If you have any questions I am in room 2) 🐠

File: frontend/src/utils.tsx (hunk 2 above)
Comment (StormTrooper-007) [reply frontend/src/utils.tsx]: Thanks pia

**Pull request merged**